report-jtreg-comparator/src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Builds.java

```diff
@@ -125,7 +126,7 @@ public static String getBuildNumber(File build) {
         }
     }
 
-    public static String getNvr(File build) {
-        return JobsRecognition.getChangelogsNvr(build);
+    public static String getNvr(File build, Options.Configuration nvrConfig) {
+        return ConfigFinder.findInConfig(new File(build, nvrConfig.getConfigFileName()), "nvr", nvrConfig.getFindQuery());
```

Comment:
maybe it would be better to have `new ConfigFinder(new File(build, nvrConfig.getConfigFileName()), "nvr", nvrConfig.getFindQuery())).findInConfig()  so I can keep searching in once created setup?

The war horse method still may be `public static` and can be called from the non static one.

Reply:
should be done now.